src/javascript/binary/base.js

```diff
@@ -222,3 +222,9 @@ if (!/backoffice/.test(document.URL)) { // exclude BO
         LocalStore.set('active_loginid', match);
     });
 }
+
+
+//this method is to prevent caching on sensitive data
+pjax_config_page("new_account/real", function(){
+    $("#secretanswer").attr("autocomplete", "off");
```

Comment:
I'd wished to, but this form is generated by perl entirely
using our in-house perl module called ***HTML::FormBuilder***, and apparently it does not support adding attribute and not much people  have related knowledge on the module, so the effort to get this up would be considerably higher than JS approach.

Is there any security concerns if we use JS? If yes, I would try to fix it from HTML